design/design-doc-stacks.md

```diff
@@ -440,7 +446,12 @@ spec:
     spec:
       containers:
       - name: sample-stack-controller
-        image: crossplane/sample-stack:latest
+        # The `image:` field is optional. If omitted, it will be
+        # filled in by the stack manager, using the same image
+        # name and tag as the package on the StackInstall object.
```

Comment:
You could mention `source` too.

Reply:
Hmm, that's true. I feel like it warrants a slightly separate paragraph

Reply:
I added a new paragraph in the text segment of this section